gdx/src/com/badlogic/gdx/maps/tiled/BaseTmxMapLoader.java

```diff
@@ -356,6 +357,8 @@ protected void loadObject (TiledMap map, MapObjects objects, Element element, fl
 					object = new PolylineMapObject(polyline);
 				} else if ((child = element.getChildByName("ellipse")) != null) {
 					object = new EllipseMapObject(x, flipY ? y - height : y, width, height);
+				} else if ((child = element.getChildByName("point")) != null) {
+					object = new PointMapObject(x, y);
```

Comment:
Does this y position need to potentially flip based on the flipY option?

Reply:
Probably not. It's a 2D coordinate, 100 from top to bottom its the same as bottom to top.

Reply:
I'm not sure why it being a 2D coordinate means it wouldn't need to flip (just like the Y position of other objects).

In a map that is 100x100, a point at say (10,10) isn't going to be correct when the Y-axis when flipY.

Reply:
In my tests, it doesn't make any difference.

Reply:
Okay, as long as it's been tested - it's been a long time since I've looked at this particular code so your understanding is probably better than mine at this point.

Reply:
Hmm, EllipseMapObject directly above the highlighted lines does use different y when flipY is true. What's different about a PointMapObject? Also, do the existing libGDX tests cover point objects? There's no new tests here.

Reply:
height variable in this context is the height of the object itself, which is zero for a point. That's why flipping is not necessary here.
About tests, i agree, it would be better to improve the existing [TiledMapObjectLoadingTest](https://github.com/libgdx/libgdx/blob/master/tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectLoadingTest.java)

Reply:
I just took a look - the global flip for y happens earlier in the method. The flip for EllipseMapObject (and friends) is to re-orient the object when accounting for the height of it (and the fact Rectange, Ellipse are bottoem-left origin) - if I am reading/recalling correctly.

Reply:
> height variable in this context is the height of the object itself, which is zero for a point. That's why flipping is not necessary here.
> About tests, i agree, it would be better to add it to the existing one: [TiledMapObjectLoadingTest](https://github.com/libgdx/libgdx/blob/master/tests/gdx-tests/src/com/badlogic/gdx/tests/TiledMapObjectLoadingTest.java)

Done in 23a405b